akka-remote-tests/src/test/scala/akka/remote/artery/ArteryFailedToBindSpec.scala

```diff
@@ -0,0 +1,38 @@
+package akka.remote.artery
+
+import akka.actor.ActorSystem
+import com.typesafe.config.ConfigFactory
+import org.scalatest.{ Matchers, WordSpec }
+
+class ArteryFailedToBindSpec extends WordSpec with Matchers {
+
+  "an ActorSystem" must {
+    "not start if port is taken" in {
+      val config = ConfigFactory.parseString(
+        s"""
+           |akka {
+           |  actor {
+           |    provider = remote
+           |  }
+           |  remote {
+           |    artery {
+           |      enabled = on
+           |      canonical.hostname = "127.0.0.1"
+           |      canonical.port = 25530
```

Comment:
instead of hard coding this (and risking conflict with something else) you can use `SocketUtil.temporaryServerAddress` or `SocketUtil.temporaryLocalPort`

Reply:
Ah that's better